dpnp/dpnp_iface_mathematical.py

```diff
@@ -327,6 +328,47 @@ def copysign(x1, x2, **kwargs):
     return call_origin(numpy.copysign, x1, x2, **kwargs)
 
 
+def cross(x1, x2, **kwargs):
+    """
+    Return the cross product of two (arrays of) vectors.
+
+    For full documentation refer to :obj:`numpy.cross`.
+
+    Limitations
+    -----------
+        Parameters ``x1`` and ``x2`` are supported as :obj:`dpnp.ndarray`.
+        Keyword arguments ``kwargs`` are currently unsupported.
+        Otherwise the functions will be executed sequentially on CPU.
+        Input array data types are limited by supported DPNP :ref:`Data types`.
+        Sizes of input arrays are limited by ``x1.size == 3 and x2.size == 3``.
+        Shapes of input arrays are limited by ``x1.shape == (3,) and x2.shape == (3,)``.
```

Comment:
```suggestion
        Parameters ``x1`` and ``x2`` are supported as :obj:`dpnp.ndarray`.
        Keyword arguments ``kwargs`` are currently unsupported.
        Sizes of input arrays are limited by ``x1.size == 3 and x2.size == 3``.
        Shapes of input arrays are limited by ``x1.shape == (3,) and x2.shape == (3,)``.
        Otherwise the functions will be executed sequentially on CPU.
        Input array data types are limited by supported DPNP :ref:`Data types`.
```